datalad/distribution/update.py

```diff
@@ -167,12 +183,13 @@ def __call__(name=None, dataset=None,
                     # For now, just use the same name
                     cmd_list.append(repo.get_active_branch())
 
-                out, err = repo._git_custom_command('', cmd_list)
-                lgr.info(out)
+                std_out, std_err = repo._git_custom_command('', cmd_list)
+                lgr.info(std_out)
                 if knows_annex(repo.path):
                     # annex-apply:
                     lgr.info("Updating annex ...")
-                    out, err = repo._git_custom_command('', ["git", "annex", "merge"])
-                    lgr.info(out)
+                    std_out, std_err = repo._git_custom_command(
+                        '', ["git", "annex", "merge"])
```

Comment:
We should move this into a `AnnexRepo.merge()` method to have things in one place.